fbw-a380x/src/systems/instruments/src/FCU/Components/NdData.tsx

```diff
@@ -33,6 +42,15 @@ export class NdData extends DisplayComponent<NdDataProps> {
     NavAidMode.Off,
   );
 
+  private lightsTest = Subject.create(false);
+
+  onAfterRender(_node: VNode): void {
+    this.sub
+      .on('ovhd_ann_lt_test_active')
+      .whenChanged()
+      .handle((v) => this.lightsTest.set(v));
+  }
+
   render(): VNode | null {
     return (
```

Comment:
I would recommend just using the same method as the navaid images for all of these.

Reply:
so using the map function? I'll give it a try again. I tried a few different methods and had trouble getting it to work but I think understand things a bit better now.